OracleDatabase/RAC/OracleDNSServer/dockerfiles/latest/named.empty

```diff
@@ -0,0 +1,10 @@
+$TTL 3H
```

Comment:
The `named` package should install this file. You should not provide it yourself.